design-system/src/design-tokens.ts

```diff
@@ -273,6 +324,27 @@ const designTokens = {
   placeholderFontColorForInput:
     'var(--placeholder-font-color-for-input, hsl(0, 0%, 60%))',
   fontSizeForInput: 'var(--font-size-for-input, 1rem)',
+  fontSizeForTextAsH1: 'var(--font-size-for-text-as-h-1, 1.5rem)',
```

Comment:
I would expect the name to be `-h1` and not `-h-1`. Might want to look into the script.